docs/guides/high-availability/concepts-multi-site.adoc

```diff
@@ -107,19 +102,11 @@ The statement "`No data loss`" depends on the setup not being degraded from prev
 
 == Known limitations
 
-Upgrades::
```

Comment:
Do we not want to mention this for Upgrades?

Reply:
Do you mean adding a note about highlighting that this is no longer the casein the `changes-26_0_0.adoc` doc?

I removed this section as session data will no longer be lost on upgrades as the `persistent-user-sessions` flag is always enabled when using `multi-site`.

Reply:
Yup, it should be highlighted in the appropriate place, probably release notes, as the `no longer be lost` part will only be true for Upgrades post KC26.